web/themes/custom/digital_gov/templates/node/node--community--teaser.html.twig

```diff
@@ -0,0 +1,69 @@
+{% set title = node.label %}
+{% set summary = content.field_summary|field_value %}
+{% set member_count = content.field_members|field_value %}
+{% set logo = content.field_dg_logo.0['#entity']|file_uri|image_style('large') %}
+{% set dg_acronym = content.field_dg_acronym|field_value %}
+{% set redirect = content.field_redirect_to.0['#title'] %}
+{% set highlight = content.field_dg_highlight.0['#markup'] %}
+{% set page_weight = content.field_page_weight.0['#markup'] %}
+{% set url = url('entity.node.canonical', {node: node.id}) %}
+
+{# todo: if high is on and page weight is 1? #}
```

Comment:
this todo will be updated when the short post is updated to take in a community option